src/irclog2html/irclogsearch.py

```diff
@@ -172,7 +159,7 @@ def search_irc_logs(query, stats=None, where=DEFAULT_LOGFILE_PATH,
             elif event == LogParser.NICKCHANGE:
                 text, oldnick, newnick = info
             else:
-                text = unicode(info)
+                text = str(info)
```

Comment:
TBH at this point `info` should already be Unicode, so

```suggestion
                text = info
```

should work fine.

I'm not sure why the old code was doing `unicode()`.   Maybe to avoid hard errors in case I introduce a new event type that returns a tuple?

Reply:
This doesn't hurt either ... I don't know enough. The defaukt thing I would do for my projects is adding annotations everywhere.